### ISSUE TITLE
Monterey, Apple Silicon M1, and Casio MZ-2000 keyboard

### ISSUE THREAD

**Issue by tiengtructiengdong** [francoisferland/casiousbmididriver#48]
Hello!
This project is promising. However, I cannot manage to connect my MZ-2000 to my M1 mac under macOS Monterey.
It does show "USB MIDI Interface" in System Information though.

Please let me know if you are still maintaining this project.
Thank you!

**Comment by francoisferland** [comment]
Hi, I am not familiar with the MZ-2000, but from a quick search I could not confirm if it has a USB or classic MIDI interface. This driver is only meant for older USB Casio keyboards. Could you describe how do you connect the keyboard with your Mac ?

**Comment by tiengtructiengdong** [comment]
Hi.

I connected via a MIDI-USB interface cable. The MIDI setting in the keyboard has a field called "Navigate Channel", which I believe to be the output channel from which Mac receives key data. The MZ-2000 is made in 2000 anyway.

The keyboard didn't show up on Audio MIDI setup. I don't know if I missed any steps, but I mapped my instruments to the correct channel.

Here are some logs if it helps at all:

`AMFI: '/Library/Audio/MIDI Drivers/CasioUSBMIDIDriver.plugin/Contents/MacOS/CasioUSBMIDIDriver' is adhoc signed.`
`/Library/Audio/MIDI Drivers/CasioUSBMIDIDriver.plugin/Contents/MacOS/CasioUSBMIDIDriver signature not valid: -67050
`

![Screen Shot 2023-01-08 at 12 43 03](https://user-images.githubusercontent.com/29170976/211182352-0f31ed15-0aa5-4a8a-b21c-79209ead7cd2.png)

![Screen Shot 2023-01-08 at 12 42 35](https://user-images.githubusercontent.com/29170976/211182348-38d10227-b6dd-46a8-9fbf-4a3936b447db.png)

Thank you so much!
